cdap-ui/server/config/ui-settings.json

```diff
@@ -1,4 +1,4 @@
 {
   "standalone.website.sdk.download": "true",
-  "ui.debug.enabled": "true"
+  "ui.debug.enabled": "false"
```

Comment:
@tonybach yes. This allows UI to extensively log in browser console. Its not necessary for production. Its a dev only setting for us.